pkg/component/component.go

```diff
@@ -38,7 +38,7 @@ type Description struct {
 	ComponentName      string                `json:"componentName,omitempty"`
 	ComponentImageType string                `json:"type,omitempty"`
 	Path               string                `json:"source,omitempty"`
-	URLs               []urlpkg.URL          `json:"url,omitempty"`
+	URLs               urlpkg.UrlList        `json:"url,omitempty"`
```

Comment:
if this is a list shouldn't he json tag be `urlList` or `urls`?

Reply:
removed `tags`, not needed anymore